Data/Primitive/PrimArray.hs

```diff
@@ -467,6 +471,65 @@ sizeofPrimArray :: forall a. Prim a => PrimArray a -> Int
 {-# INLINE sizeofPrimArray #-}
 sizeofPrimArray (PrimArray arr#) = I# (quotInt# (sizeofByteArray# arr#) (sizeOf# (undefined :: a)))
 
+-- | Map each element of the primitive array to a monoid, and combine the results.
+--   The combination is right-associated, and the accumulation is lazy.
+--   An example:
+--
```

Comment:
Use `==== __Example__` to collapse the example. It's an experimental feature, but we use it anyway.